C#/C#.sublime-syntax

```diff
@@ -1674,13 +1674,21 @@ contexts:
       captures:
         1: keyword.control.trycatch.catch.cs
         2: meta.group.cs punctuation.section.group.begin.cs
-      push: [catch_block, var_declaration_explicit]
+      push: [catch_block, var_declaration_explicit, catch_namespace]
     - match: 'catch'
       scope: keyword.control.trycatch.catch.cs
       push: trycatch_block
     - match: (?=\S)
       pop: true
 
+  catch_namespace:
```

Comment:
I'm not versed in C#, but wouldn't it make sense to perform this kind of capture in the `var_declaration_explicit` context? Are type declarations with namespaces not valid?

Reply:
good point, I somehow hadn't noticed that the syntax was missing support for things like `System.String blah = "test";`